src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__product-image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: color 0.3s;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text-color;
+    margin: 0 0 16px;
+  }
+
+  &__product-rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin: 0 0 24px;
+
+    .reviews-count {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__product-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 16px;
+
+    .price-title {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    .price-numbers {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: 40px;
+
+    background-color: $blue-brand-color;
+    border: 1px solid $blue-brand-color;
```

Comment:
The background-color and border-color are the same. You can use shorthand `border: 1px solid` without specifying the color as it will inherit the color from the background.